src/isar/modules.py

```diff
@@ -30,6 +33,28 @@
 from robot_interfaces.robot_telemetry_interface import RobotTelemetryInterface
 
 
+class APIModule(Module):
+    @provider
+    @singleton
+    def provide_drive_to(self, scheduling_utilities: SchedulingUtilities) -> DriveTo:
+        return DriveTo(scheduling_utilities)
+
+
+    @provider
+    @singleton
+    def provide_start_mission(self,
+        mission_reader: MissionReader,
+        scheduling_utilities: SchedulingUtilities) -> StartMission:
+        return StartMission(mission_reader,scheduling_utilities)
+
+
+    @provider
+    @singleton
+    def provide_stop_mission(self,
+        queues: Queues) -> StopMission:
+        return StopMission(queues)
```

Comment:
```suggestion
    @provider
    @singleton
    def provide_drive_to(self, scheduling_utilities: SchedulingUtilities) -> DriveTo:
        return DriveTo(scheduling_utilities)

    @provider
    @singleton
    def provide_start_mission(self,
        mission_reader: MissionReader,
        scheduling_utilities: SchedulingUtilities) -> StartMission:
        return StartMission(mission_reader,scheduling_utilities)

    @provider
    @singleton
    def provide_stop_mission(self,
        queues: Queues) -> StopMission:
        return StopMission(queues)
```

Reply:
There should only be one newline between function definitions. There should be two newlines between class definitions.